### DISCUSSION TITLE
VirtualServer Host wildcard hostname

### DISCUSSION THREAD

**Discussion by chriskolenko** [nginx/kubernetes-ingress#2626] [Q&A]
Hey everyone, 

Just wanted to know the reason behind VirtualServer / VirtualServerRoutes not support wildcard hostnames.

**Comment by brianehlert** [comment]
We have this in the roadmap to resolve soon.

**Reply by chriskolenko** [reply]
Cheers. Anything I can do to help? I'm playing around with disabling the subdomain validation. I'm happy to create a test and create a PR if I can figure it out.

**Reply by brianehlert** [reply]
Always happy with folk contributing as they have time

**Reply by chriskolenko** [reply]
Let me spend some time over the weekend. I'll keep this discussion open. Will need some tips on how you wanted the validation to be done.

**Comment by jjngx** [comment]
3 years old. No contact. Closing
